modules/widgets/src/theme-widget.tsx

```diff
@@ -121,10 +121,6 @@ export class ThemeWidget implements Widget<ThemeWidgetProps> {
 
     Object.assign(this.props, props);
     this.update();
-
-    this.deck?.setProps({
```

Comment:
I would have loved to see a method `_setTheme()` that clearly shows how the theme is being set. The removeStyles/applyStyles is mixed with a bunch of other high-context logic here.